profiles/utils.py

```diff
@@ -47,3 +53,22 @@ def get_current_user():
 
 def get_current_service():
     return getattr(_thread_locals, "service", None)
+
+
+def user_has_staff_perms_to_view_profile(
+    user: "users.models.User", profile: "profiles.models.Profile"
+) -> bool:
+    """
+    Checks is passed user has "can_view_profiles" permissions
+    for any service connected to the passed profile.
+    """
+
+    service_conns = profile.service_connections.filter(enabled=True)
+    if any(
```

Comment:
Could this be just `return any(...`. Don't know about readability, but would save a few LOC 😅

Reply:
ah, right, missed that 😅